salt/beacons/swapusage.py

```diff
@@ -0,0 +1,74 @@
+"""
+Beacon to monitor swap usage.
+
+.. versionadded:: 3003
```

Comment:
Assumed `3003`.

Reply:
`3003.0`?

Reply:
We did discuss this in the open hour, but there is not a SEP currently to actually change this. I'll bring it up again to see which release we want to start implementing this and when we can get a SEP prepped. For now you can keep it at 3003 and we can change later if needed.